docs/readmes/proposals/p010_vendor_neutral_dp.md

```diff
@@ -429,6 +429,40 @@ scope of work.
 | MS6: Testing and bug-fixing with a small pilot deployment (3-5 CBSDs) |          |          |          |          | x        | x        | x        |
 | MS7: Documentation and beta deployment support gate          |          |          |          |          |          |          | x        |
 
+
+# Implementation
+## Phase 1 (Passive Mode / Proxy Mode)
+NOTE: In the Passive mode Domain Proxy should proxy messages from CBSDs to SAS without any changes besides batching of messages outgoing to SAS. 
+### System Architecrue
+
+<img src="media/dp_phase_1_hl_arch.png" style="width:5.54167in;height:2.72917in" />
+
+This Architecture assumes following:
+1) we are going to talk to CBSDs which are "speaking" different procols (not a differen version of the same protocol)
+1) we are going to have different number of CBSDs per protocol
+1) CBSD is going to send synchronous REST API calls to DP, and DP is going to send synchronous REST API calls to SAS
+1) Rest of the communications inside DP can be synchronous or asynchronous, as long as it satisfies previous statement
+
+#### Key Components:
+Protocol Controller(PC) - µService responsible for translation CBSD protocol to internal protocol(SAS friendly protocol) and backwards. 
+Radio Controller(RC): µService responsible for state management using it's SQL DB(RC_DB). State in this case is some form of requests from CBSDs to SAS and responses from SAS to CBSDs. Specific DataStructures to be defined yet.
```

Comment:
In Magma we model state not requests (see level vs edges), so will be good to be explicit about the type of state that gets maintained here.